src/esp/physics/objectWrappers/ManagedArticulatedObject.h

```diff
@@ -81,6 +90,13 @@ class ManagedArticulatedObject
     return {};
   }
 
+  int getLinkIdFromName(const std::string& _name) const {
+    if (auto sp = getObjectReference()) {
+      return sp->getLinkIdFromName(_name);
+    }
+    return -1;
```

Comment:
This is a valid value for the base link. Is it intended to be a null return or default to base link when the name doesn't exist?

Reply:
Hmm, this isn't for when the link doesn't exist, it's for when the object doesn't exist (sp is what is the wrapper around the object)

Usually these values on return were valid but empty. Not sure what to do here.

Reply:
Yeah, I guess if this object is invalid other parts of the API should be crashing, so probably not a problem here with the default return.